requirements.in

```diff
@@ -0,0 +1,17 @@
+#-----------------------------------
+#   Utils
+#-----------------------------------
+funding-service-design-utils<=1.1.0
+
+cssmin==0.2.0
+email_validator==2.0.0.post2
+flask-assets==2.0
+flask-compress==1.13
+flask-limiter==3.3.0
+flask-talisman==1.0.0
+flask==2.3.1
+govuk-frontend-jinja==2.6.0
+govuk-frontend-wtf==2.4.0
+gunicorn==20.1.0
+jsmin==3.0.1
+redis==4.5.4
```

Comment:
Please remove any dependencies we're not using, certainly would include redis and maybe some others.

Reply:
I have removed the extraneous dependencies.